x-pack/plugins/synthetics/server/saved_objects/synthetics_param.ts

```diff
@@ -10,6 +10,7 @@ import { syntheticsParamType } from '../../common/types/saved_objects';
 export const SYNTHETICS_SECRET_ENCRYPTED_TYPE = {
   type: syntheticsParamType,
   attributesToEncrypt: new Set(['value']),
+  attributesToIncludeInAAD: new Set(['key', 'description', 'tags']),
```

Comment:
This object's attributes are dynamically mapped, and there was not exclude list. This is somewhat dangerous. I based the include list on the result of creating a synthetics global parameter in the UI.